src/integrations/prefect-dbt/prefect_dbt/cli/commands.py

```diff
@@ -398,6 +398,7 @@ def _compile_kwargs(self, **open_kwargs: Dict[str, Any]) -> Dict[str, Any]:
 
 
 @task
+@sync_compatible
```

Comment:
The `@sync_compatible` needs to come first for the dispatch to work correctly.
```suggestion
@sync_compatible
@task
```

Reply:
Huh, interesting!